### ISSUE TITLE
Style/AccessorGrouping + Layout/LineLength make impossible to represent some data structures without violating Rubocop default rules

### ISSUE THREAD

**Issue by Fire-Dragon-DoL** [rubocop-hq/rubocop#8978]
The default configuration of:
- `Layout/LineLength` set to `120`
- `Style/AccessorGrouping` set to `grouped`

## Expected behavior

I expected the Rubocop default configuration to allow representation of any data structure without violating the guidelines.

## Actual behavior

The default configuration of:
- `Layout/LineLength` set to `120`
- `Style/AccessorGrouping` set to `grouped`

Requires all the attributes of a data structure to be specified on a single line of maximum 120 characters.
This makes impossible representing some data structures (especially user-provided information, such as "address" data structures) because the list of the attributes can't fit in 120 characters.

Notice that the offense is auto correctable and **when corrected, the offense is still present.**

#### Version 1

```ruby
# frozen_string_literal: true

# Doc
class Address
  attr_accessor :name, :unit_number, :street_name, :city, :province, :postal_code, :location, :direction, :country, :region
end
```

#### Version 2

```ruby
# frozen_string_literal: true

# Doc
class Address
  attr_accessor :name, :unit_number, :street_name, :city, :province, :postal_code, :location, :direction
  attr_accessor :country, :region
end
```

## Steps to reproduce the problem

Put the above examples inside 2 ruby files and run `rubocop` on both. The example will provide errors in both cases, the output is provided in this issue.

### Output 1

```
$ rubocop /Users/francesco/Desktop/address1.rb
Inspecting 1 file
C

Offenses:

Desktop/address1.rb:5:121: C: Layout/LineLength: Line is too long. [123/120]
  attr_accessor :name, :unit_number, :street_name, :city, :province, :postal_code, :location, :direction, :country, :region
                                                                                                                        ^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

### Output 2

```
$ rubocop /Users/francesco/Desktop/address2.rb
Inspecting 1 file
C

Offenses:

Desktop/address2.rb:5:3: C: Style/AccessorGrouping: Group together all attr_accessor attributes.
  attr_accessor :name, :unit_number, :street_name, :city, :province, :postal_code, :location, :direction
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Desktop/address2.rb:6:3: C: Style/AccessorGrouping: Group together all attr_accessor attributes.
  attr_accessor :country, :region
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 2 offenses detected, 2 offenses auto-correctable
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ rubocop -V
1.1.0 (using Parser 2.7.2.0, rubocop-ast 1.0.1, running on ruby 2.6.6 x86_64-darwin19)
```

**Comment by koic** [comment]
For example, the warning can be suppressed by the following newline wrapping. It will be helpful to notice with auto-correction.

```ruby
class Address
  attr_accessor :name, :unit_number, :street_name, :city, :province, :postal_code, :location, :direction, :country,
                :region
end
```

**Comment by Fire-Dragon-DoL** [comment]
Interesting suggestion @koic , if that's the case I guess the bug should be targeted at the autocorrector part, instead
